dotnet-client-libraries/HandlingDatesWithNodaTime/Tests/NodaTimeUnitTest.cs

```diff
@@ -0,0 +1,120 @@
+﻿using NodaTime;
+using NodaTime.Text;
+
+namespace Tests;
+
+public class NodaTimeUnitTest
+{
+    [Fact]
+    public void WhenGetCurrentInstantMethodIsCalled_ThenReturnCurrentInstant()
+    {
+        var instant = SystemClock.Instance.GetCurrentInstant();
+
+        Assert.True(instant != default(Instant));
+    }
+
+    [Fact]
+    public void WhenFromUnixTimeMethodIsCalled_ThenReturnCurrentInstant()
```

Comment:
The name of the test is incorrect as the code is not returning the current instant, but rather creating a specific instant based on UTC definition.

Reply:
This is updated, please review